pynest/nest/__init__.py

```diff
@@ -82,27 +87,42 @@ def catching_sli_run(cmd):
         message = sli_pop()
         commandname = sli_pop()
         engine.run('clear')
-        raise _kernel.NESTError("{0} in {1}{2}".format(errorname, commandname, message))
+        raise _kernel.NESTError("{0} in {1}{2}".format(
+            errorname, commandname, message))
 
 sli_run = hl_api.sr = catching_sli_run
 
+
 def sli_func(s, *args, **kwargs):
-    """This function is a convenience function for executing the 
-       sequence sli_push(args); sli_run(s); y=sli_pop(). It takes
-       an arbitrary number of arguments and may have multiple
-       return values. The number of return values is determined by
-       the SLI function that was called.
-
-       Keyword arguments:
-       namespace - string: The sli code is executed in the given SLI namespace.
-       litconv   - bool  : Convert string args beginning with / to literals.
-       
-       Examples:
-         r,q = sli_func('dup rollu add',2,3)
-         r   = sli_func('add',2,3)
-         r   = sli_func('add pop',2,3)
-         l   = sli_func('CreateLayer', {...}, namespace='topology')
-         opt = sli_func('GetOptions', '/RandomConvergentConnect', litconv=True)
+    """Convenience function for executing the sequence
+    sli_push(args); sli_run(s); y=sli_pop().
+
+    The function takes an
```

Comment:
Something is missing here. Probably, the sentence can just be deleted, as the thing with the arbitrary number of arguments is written below.